main.py

```diff
@@ -0,0 +1,147 @@
+import json
+import socket
+import threading
+from select import select
+from typing import Dict, List, Optional, Union, Any
```

Comment:
```suggestion
from typing import Any
```
используем встроенное